marketplace/deployer_helm_tiller_base/test/post-install.yaml

```diff
@@ -0,0 +1,57 @@
+apiVersion: v1
```

Comment:
Requiring partners to supply these two hooks are really not ideal. We can automatically create a wrapper chart (either at docker build time or at run time). See [this example](https://github.com/GoogleCloudPlatform/marketplace-k8s-app-tools/blob/huyhg/doc/d/docs/building-deployer-helm.md#part-1b-recommended-creating-a-wrapper-chart) on how wrapper chart works.

There is one wrinkle: if the user tries to manipulate the release with helm directly, they will need access to the wrapper chart source. This encourages creating the wrapper chart at build time, so that the end user can just download the source code for it directly from the container.

The nice thing about helm chart is that in its `Chart.yaml` there is instruction on how to fetch a different version of the chart. So the main chart in this case should still have that declared properly, and the user after downloading the wrapper chart can still upgrade the wrapped main chart. Or they can just hand edit the thing.

Reply:
I have some hesitations about marketplace generating the wrapper chart automatically, but having the partner model their deployer as a wrapper chart itself is definitely cleaner.

Let's discuss if you feel strongly about marketplace generating the wrapper chart. :)